snow/networking/throttler/message_throttler.go

```diff
@@ -9,11 +9,12 @@ import (
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/constants"
 	"github.com/ava-labs/avalanchego/utils/logging"
 )
 
 const (
-	defaultIntervalsUntilPruning uint32 = 60
+	defaultIntervalsUntilPruning uint32 = 5
```

Comment:
What's the rationale for this? I believe we only need to remove spenders from memory to avoid a memory leak, but I'm not sure there's a need to remove them earlier

Reply:
Bumped it to 10 so it should remove after 150 seconds which seems reasonable